cmd/zoekt-sourcegraph-indexserver/index_test.go

```diff
@@ -246,6 +247,8 @@ func TestIndex(t *testing.T) {
 		},
 	}}
 
+	logger := log.NoOp()
```

Comment:
I tested a Scoped logger here initially, in the case where we want structured logging output during test runs. However, at this time I don't think there's a need and instead it would potentially just clutter the output.

We can revisit this on test by test basis.

Reply:
Also we could make logger an argument for each test case and call it below as tc.logger but as of now the shared logger instance simplifies the code.

Reply:
you can use `logtest.Scoped(t)` then if the test fails it will include the logs.

Reply:
Updated

Reply:
Updated